src/main/java/com/conveyal/datatools/editor/datastore/FeedTx.java

```diff
@@ -63,7 +63,8 @@
     public NavigableSet<Tuple2<String, String>> tripPatternsByStop;
 
     /** number of schedule exceptions on each date - this will always be null, 0, or 1, as we prevent save of others */
-    public ConcurrentMap<LocalDate, Long> scheduleExceptionCountByDate;
+    // This has been removed because it has been found to cause corruption for mapdb for some unknown reason.
```

Comment:
We should be a bit more clear about what "this" is - we actually know more about the situation than is communicated in the comments. It's specifically that when shutting down the JVM, most MapDB tables seem to be closed cleanly, but this particular one (and perhaps others created with the bindHistogram method?) are not put into a clean state.